docs/config.rst

```diff
@@ -70,6 +70,9 @@ Default configuration
     # Crop each capture border in order to fit the paper size
     captures_cropping = False
 
+    # Thick (in pixels) between captures and picture borders/texts
+    margin_thick = 100
```

Comment:
I have an issue with this setting, if I write 1000 ou 2000, the pibooth crashes...

```
pibooth/pibooth/pictures/factory.py", line 451, in _image_resize_keep_ratio
    image = cv2.resize(cropped, (max_w, max_h), interpolation=inter)
cv2.error: OpenCV(4.1.1) /tmp/opencv-20190901-52549-14x6tkx/opencv-4.1.1/modules/imgproc/src/resize.cpp:3723: error: (-215:Assertion failed) inv_scale_x > 0 in function 'resize'
```

Reply:
Will be corrected by #135